packages/edit-navigation/src/utils/fetch-link-suggestions.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { addQueryArgs } from '@wordpress/url';
+/**
+ * External dependencies
+ */
+import { flatten, slice, map, flow } from 'lodash';
+import { decodeEntities } from '@wordpress/html-entities';
+import { __ } from '@wordpress/i18n';
+
+const mapC = ( f ) => ( arr ) => map( arr, f );
```

Comment:
The naming conventions in this file conflict with the code style guidelines.
https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#naming-conventions

In particular I'm thinking about:
- the `c` in `mapC`
- `f`
- `arr`
- `Obj`
- `Req`